upload/extension/opencart/admin/controller/module/banner.php

```diff
@@ -72,6 +72,12 @@ public function index(): void {
 		} else {
 			$data['items'] = 4;
 		}
+		
+		if (isset($module_info['mobileitems'])) {
+			$data['mobileitems'] = $module_info['mobileitems'];
+		} else {
+			$data['mobileitems'] = 1;
```

Comment:
can u add 1 to constant like

private const  DEFAULT_MOBILE_ITEMS = 1

Reply:
I did not understand the need for that, as it is a starting point for the number 1 to appear in the settings, and it is kept as the minimum with one banner, or it is modified according to desire and preservation

Reply:
if opencart 4 start use php 81 we need try do more readable code